rasa/constants.py

```diff
@@ -59,3 +59,7 @@
 
 DEFAULT_SESSION_EXPIRATION_TIME_IN_MINUTES = 60
 DEFAULT_CARRY_OVER_SLOTS_TO_NEW_SESSION = True
+
+ENV_GPU_CONFIG = "TF_GPU_MEMORY_ALLOC"
```

Comment:
what about `TF_FORCE_GPU_ALLOW_GROWTH`?

Reply:
@wochinge It comes from tensorflow, so we don't need to add it here.